turtle_control:
  angular_velocity: 0.3 
  linear_speed: 0.5        
  trajectory:
    radius: 1.0           
    center_x: 5.5         
    center_y: 5.5        
  rate: 10                
  color:                   
    r: 255
    g: 0
    b: 0
